stream-chat-android-ui-uitests/src/main/java/io/getstream/chat/android/uitests/app/login/LoginActivity.kt

```diff
@@ -0,0 +1,210 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Icon
+import androidx.compose.material.Switch
+import androidx.compose.material.Text
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.compose.ui.components.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.ui.channel.ChannelListActivity
+import io.getstream.chat.android.uitests.R
+import io.getstream.chat.android.uitests.app.compose.ChannelsActivity
+
+/**
+ * An Activity that allows users to log in using one of our predefined sample users.
+ */
+class LoginActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            ChatTheme {
+                LoginScreen()
+            }
+        }
+    }
+
+    @Composable
+    fun LoginScreen() {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(modifier = Modifier.height(32.dp))
+
+            Icon(
+                modifier = Modifier.size(width = 80.dp, height = 40.dp),
+                painter = painterResource(id = R.drawable.ic_stream),
+                contentDescription = null,
+                tint = ChatTheme.colors.primaryAccent,
+            )
+
+            Spacer(modifier = Modifier.height(20.dp))
+
+            val checkedState = remember { mutableStateOf(false) }
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Switch(
```

Comment:
The design leaves much to be desired. My plan is to add 2 tiles instead of one switch. The selected tile would have an outline.